tests/test_userstory_39.py

```diff
@@ -56,3 +62,87 @@ def test_melding_overlast():
         'object': verblijfsobject['url'],
     })
     assert 'url' in zaak_object
+
+
+    '''
+    Create temporary files to test file uploading
+    '''
+    # Text file (can't be empty or the API throws an error)
+    fd, txt_path = tempfile.mkstemp()
+    with open(txt_path, 'w') as tmp:
+        tmp.write('some additional information')
+
+    # Image (png format)
+    im_format = 'png'
+    im = Image.new('RGB', (512, 512), 'blue') #plain blue square
+    buffered = BytesIO()
+    im.save(buffered, format=im_format)
+
+    '''
+    Upload the files with POST /enkelvoudiginformatieobject (DRC)
+    '''
+
+    text_attachment = drc_client.create_form('enkelvoudiginformatieobject', {
+    'identificatie': uuid.uuid4().hex,
```

Comment:
code formatting not following pep8

Reply:
added indentation

Reply:
pep8 config in `setup.cfg` so that we can run `pep8` on the cli would be great

Reply:
also used the same config I found on other setup.cfg, and corrected some pep8 violations